romancal/datamodels/schemas/reference_files/flat.schema.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/flat.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: ../core.schema
+- type: object
+  properties:
+    data:
+      title: The science data
+      fits_hdu: SCI
```

Comment:
Can the `fits_hdu` property here and in the other arrays can be removed?  That's a pseudo-validator that defines a mapping between an ndarray model attribute and a FITS HDU.  I didn't think Roman was using that feature of DataModel.

Reply:
Done.